app/Jobs/UpdateWikiMetricDailyJob.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Wiki;
+use App\WikiDailyMetric;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+
+class UpdateWikiMetricDailyJob implements ShouldQueue
```

Comment:
I totally get how we go there but I wonder if we should think a little harder about the names and clarifying how this job is different from all the other confusing metric jobs